test/vendor/tracekit.test.js

```diff
@@ -119,8 +119,8 @@ describe('TraceKit', function(){
             // we can't do that without clobbering TraceKit's handler, which can only
             // be installed once.
             var oldOnError = window.onerror;
-            window.onerror = function(message) {
-                if (message == testMessage) {
+            window.onerror = function(message, url, lineNo) {
+                if (message == testMessage || lineNo === 1337) {
```

Comment:
So, this relied on passing a unique message to prevent bubbling to mocha's onerror ...

But, my tests send custom messages. So I made it additionally accept a unique integer.